### DISCUSSION TITLE
network/modules/metallb/install/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2033] [Announcements]
# network/modules/metallb/install/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/network/modules/metallb/install/

**Comment by san3Xian** [comment]
在 kube-proxy ipvs 模式下，用 metallb ，需要打开 kube_proxy_strict_arp. 
否则会导致VIP被多个节点arp reply，影响流量  
https://metallb.universe.tf/installation/  
建议加到文档中，并标记为需要注意的配置
